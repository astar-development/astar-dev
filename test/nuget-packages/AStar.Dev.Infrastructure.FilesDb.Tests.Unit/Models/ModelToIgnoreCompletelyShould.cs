using AStar.Dev.Infrastructure.FilesDb.Models;

namespace AStar.Dev.Infrastructure.FilesDb.Tests.Unit.Models;

public sealed class ModelToIgnoreCompletelyShould
{
    [Fact]
    public void ReturnTheExpectedToStringOutput()
        => new ModelToIgnore(){UpdatedOn = new DateTimeOffset(2025,1,2,3,4,5, new TimeSpan(0, 0, 0))}.ToString().ShouldMatchApproved();
}
