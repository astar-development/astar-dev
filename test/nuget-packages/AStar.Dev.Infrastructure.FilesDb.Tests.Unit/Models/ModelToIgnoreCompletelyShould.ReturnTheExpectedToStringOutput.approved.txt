{
  "id": 0,
  "value": "",
  "updatedBy": "Jay Barden",
  "updatedOn": "2025-01-02T03:04:05+00:00"
}